server/historian/packages/historian-base/src/utils.ts

```diff
@@ -67,14 +68,35 @@ export function getRequestErrorTranslator(
         if (typeof error === "number" || !Number.isNaN(Number.parseInt(error, 10)))  {
             const errorCode = typeof error === "number" ? error : Number.parseInt(error, 10);
             winston.error(getStandardLogErrorMessage(`${errorCode}`));
+            Lumberjack.log(getStandardLogErrorMessage(`${errorCode}`), LogLevel.Error);
             throw new NetworkError(
                 errorCode,
                 "Internal Service Request Failed",
             );
         }
         // Treat anything else as an internal error, but log for debugging purposes
         winston.error(getStandardLogErrorMessage(safeStringify(error)));
+        Lumberjack.log(getStandardLogErrorMessage(safeStringify(error)), LogLevel.Error);
```

Comment:
In this case, do we want to explicit pass error as an individual parameter, instead of stringifying it?